packages/web-components/src/select/select.stories.ts

```diff
@@ -0,0 +1,84 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../helpers.stories.js';
+import { SelectSize as SelectSetSize } from './select.options.js';
+import type { Select as FluentSelect } from './select.js';
+import './define.js';
+
+type SelectStoryArgs = Args & FluentSelect;
+type SelectStoryMeta = Meta<SelectStoryArgs>;
+
+const storyTemplate = html<SelectStoryArgs>`
+  <fluent-select
+    control-size="${x => x.controlSize}"
+    ?disabled="${x => x.disabled}"
+    ?required="${x => x.required}"
+    ?autofocus="${x => x.autofocus}"
+  >
+    <label slot="label">${x => x.label}</label>
+    <option value="option1">Option 1</option>
+    <option value="option2">Option 2</option>
+    <optgroup label="Group 1">
+      <option value="option3">Option 3</option>
+      <option value="option4">Option 4</option>
+    </optgroup>
+  </fluent-select>
+`;
+
+export default {
+  title: 'Components/Select',
+  args: {
+    disabled: false,
+    label: 'Select an option',
+    controlSize: SelectSetSize.medium,
+    required: false,
+    autofocus: false,
+  },
+  argTypes: {
+    label: { control: 'text' },
+    disabled: { control: 'boolean' },
+    controlSize: {
+      control: {
```

Comment:
Storybook should show kebab-case attribute.
```suggestion
      name: 'control-size',
      control: {
```